src/basic/virt.c

```diff
@@ -156,6 +156,7 @@ static Virtualization detect_vm_dmi_vendor(void) {
                 Virtualization id;
         } dmi_vendor_table[] = {
                 { "KVM",                 VIRTUALIZATION_KVM       },
+                { "OpenStack Nova",      VIRTUALIZATION_KVM       },
```

Comment:
https://cloudinit.readthedocs.io/en/latest/topics/datasources/openstack.html

Hmm, that suggests "OpenStack Compute" is also something used?

Does openstack document the used strings anywhere?

Reply:
The document says that in x86 architecture , `openstack Nova` and `openstack compute` can be used to determine whether it is openstack. 
But I'm not sure if arm architecture has a product name `OpenStack Compute`

Reply:
It's better that use `OpenStack `instead of `OpenStack Nova` or `OpenStack Compute`